spec.html

```diff
@@ -24131,7 +24142,7 @@ <h1>parseFloat ( _string_ )</h1>
         1. If neither _trimmedString_ nor any prefix of _trimmedString_ satisfies the syntax of a |StrDecimalLiteral| (see <emu-xref href="#sec-tonumber-applied-to-the-string-type"></emu-xref>), return *NaN*.
         1. Let _numberString_ be the longest prefix of _trimmedString_, which might be _trimmedString_ itself, that satisfies the syntax of a |StrDecimalLiteral|.
         1. Let _mathFloat_ be MV of _numberString_.
```

Comment:
Should/could this be v(numberString)?

Reply:
Well, there's a conversion a few lines down, "1. Return the Number value for _mathFloat_." I thought it would make sense to leave a variable called "math" a mathematical value, though it'd also be valid to perform the conversion a few lines higher. Do you have an opinion on these decisions?

Reply:
I’m asking about using the new shorterhand for MV as opposed to “MV of”

Reply:
`v(foo)` isn't a shorthand for `MV of foo`.

MV is a syntax-directed operation that converts a sequence of code points into a mathematical value. Overstruck-v converts a Number into a mathematical value.

Reply:
Ah, thanks for clarifying.